src/main/java/modbat/mbt/IllegalArgumentException.java

```diff
@@ -0,0 +1,7 @@
+package modbat.mbt;
```

Comment:
Please remove this class. There already exists java.lang.IllegalArgumentException.
This is why I suggested that name. :-)